CMakeLists.txt

```diff
@@ -1,11 +1,22 @@
-project(HighFive)
 cmake_minimum_required(VERSION 3.0)
 
-enable_testing()
+if(POLICY CMP0048)
+  cmake_policy(SET CMP0048 NEW)  # package version behavior added in cmake 3.0
```

Comment:
I do not understand why you enable these policies explicitly. To my understanding, they are already enabled if available.

Reply:
I would recommend following [my advice here](https://cliutils.gitlab.io/modern-cmake/chapters/basics.html), and doing something like this:

```cmake
cmake_minimum_required(VERSION 3.0)

if(${CMAKE_VERSION} VERSION_LESS 3.13)
    cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
else()
    cmake_policy(VERSION 3.13)
endif()
``` 

Then test it with the highest version listed (3.13) and make sure it works. When new CMakes come out, you can update the number here after testing the new version. If you find you depend on any old policies, try to fix the issue. Piecemeal setting policies is not a good idea.

This is now supported in CMake as a language feature, but due to a MSVC error, keeping it written this way is probably best for now.

Reply:
Very interesting, I was not aware of this! But it seems this may require more extensive testing of cmake versions? At least I do not feel comfortable to set any specific version as supported. But please fork my PR and exchange this part?